### PR TITLE
Added support for SUSE

### PR DESCRIPTION
**Pull request by cschneemann**
Tested with SUSE Linux Enterprise 15 Servicepack 1.

### CODE REVIEW

File: resolver/map.jinja [hunk 1]
```diff
@@ -8,6 +8,9 @@
         'Debian': {
             'conf_path': '/etc/resolvconf/run/resolv.conf',
         },
+        'SUSE': {
```
Comment (myii) [inline resolver/map.jinja]:
```suggestion
        'Suse': {
```

In order to maintain consistency with other formulas, please consider the suggestion above.  Refer back to the following search for more details:

* https://github.com/search?l=HTML%2BDjango&q=org%3Asaltstack-formulas+SUSE&type=Code

Comment (myii) [reply resolver/map.jinja]: @cschneemann Of course, please confirm that works at your end first.

**Comment by cschneemann** [comment]
@myii the requested change is of course better, it should be consistent everywhere.

**Comment by cschneemann** [comment]
And of course: tested and still working :)

**Comment by myii** [comment]
Merged, thanks @cschneemann!

**Comment by cschneemann** [comment]
> Merged, thanks @cschneemann!

Thanks for the quick response/merge

**Comment by myii** [comment]
You're welcome, if only every PR was this easy to review!

**Pull request merged**